pleroma/emoji-index/QueerCats/README.md

```diff
@@ -0,0 +1,8 @@
+# Deploy Emojis To Pleroma
+
+``` bash
+
+MIX_ENV=prod mix pleroma.emoji ls-packs -m https://raw.githubusercontent.com/mcrosson/QueerCats/pleroma-emoji-pack/pleroma/emoji-index/QueerCats/index.json
```

Comment:
Is there a reason this points to your fork? If I setup a github pages for this repo we should be able to link to the json there right?

Reply:
Correct, those were placeholders I used for testing and I didn't want to break the links when I submitted the PR in case others were testing.